src/utils.js

```diff
@@ -1,5 +1,31 @@
-const fetchWeather = cityName => {
-  const API_KEY = "<INSERT_API_KEY>";
-  // TODO: fetch weather forecast from endpoint
-  // from https://openweathermap.org/api
-};
+import axios from 'axios';
+export const fetchWeather = async (cityName, unit) => {
+    // const API_KEY = "d29e9cce44012ae03806fcd9edc39a4e";
+    const API_KEY = "08b88cc8686401a7f16cf2b1f37c2464";
```

Comment:
[question] How might you approach adding an API key but not committing it in the code for this JavaScript app?

Reply:
I would setup environment variables.